tests/security/audit/autrace.pm

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Verify the "autrace" utility traces individual processes in a fashion similar to strace.
+#          The output of autrace is logged to the audit log.
+# Maintainer: llzhao <llzhao@suse.com>, shawnhao <weixuan.hao@suse.com>
+# Tags: poo#81772
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub get_pid {
+    # Call this fuction to extract pid from autrace output in command line
+    my $tmp_output = '/tmp/out';
+    my $tmp_backup = '/tmp/backup';
+    script_run("tail -1 /tmp/out > $tmp_backup");
+    script_run("awk -F\\' '{print \$2}' $tmp_backup > $tmp_output");
+    my $pid = script_output("cat $tmp_output | cut -d ' ' -f 4");
+    return ($pid);
+}
+
+sub run {
+    my $audit_log = '/var/log/audit/audit.log';
+    my $tmp_output = '/tmp/out';
+
+    select_console 'root-console';
+
+    # Install related packages and start audit service
+    zypper_call('in audit libaudit1');
+    assert_script_run("systemctl start auditd");
```

Comment:
I see that there are some statement using (" ") and some (' ') , I think currently we mostly use the (' '), except for we will change to  (" ") if some $variable used.

Reply:
Oh! Good point! I was wondering the differences between using "" and '' when coding. I will change them if no variables used.